### PR TITLE
chore: Local bucketing proxy documentation

### PR DESCRIPTION
**Pull request by JamieSinn**

### CODE REVIEW

**Review by jonathannorris — CHANGES_REQUESTED** [review]
Can we split this off into its own docs page?

File: docs/sdk/local-bucketing-proxy/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+the deployment options and hw
+
+### Without Local Bucketing Proxy
+
+Each instance of a local bucketing SDK will connect to the Config CDN; as well as the Events API to submit events. While 
+Cloud bucketing server SDK's will connect to the Bucketing API which then handles the config retrieval and event 
+submission for variable evaluations.
+
+
+![Server SDK Topology](/server-sdk-topology.svg)
```
Comment (jonathannorris) [inline docs/sdk/local-bucketing-proxy/index.md:41]: These SVGs are very large: 
image

**Comment by chris-hoefgen** [comment]
On the SDK home page the `Local Bucket Proxy` section should be updated to be `SDK Proxy` and probably link to that set of documentation

**Review by cdruxerman — APPROVED** [review]
I've taken a few reviews of the preview site, I'm good with the current status to go live and edit further after feedback in the wild.

**Pull request merged**